high_card.rb

```diff
@@ -1,5 +1,29 @@
 # Use these two arrays to generate a deck of cards.
+require "pry"
+
+
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+deck = ranks.product(suits).shuffle
+
 
 players = []
+hands = {}
+
+
+puts players.length.to_s + " players so far. Enter a player name or type 'play'."
+
+until players.include? 'play' do
+  players << gets.chomp
+end
+
+players = players - ['play']
+
+players.each do |player|
+  hands[player] = deck.sample
+end
+
```

Comment:
I know you mentioned having trouble handling face cards, but I'd at least like to see a solution that demonstrates what you got to work with numbered cards.